.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+# vscode settings
+.vscode
```

Comment:
This is better placed in your personal .gitignore file, which you put in your home dir.